src/index.css

```
@@ -19,32 +19,50 @@ figure, figcaption, footer, header, hgroup,
 menu, nav, output, ruby, section, summary,
 time, mark, audio, video {
     margin: 0;
-	padding: 0;
-	border: 0;
-	font-size: 100%;
-	font: inherit;
-	vertical-align: baseline;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
```

Biome (1.9.4)

[error] Unexpected shorthand property font after font-size



(lint/suspicious/noShorthandPropertyOverrides)